pom.xml

```diff
@@ -48,14 +48,15 @@
     <build_cpp>OFF</build_cpp>
     <cpp_tests>OFF</cpp_tests>
     <build_arrow>OFF</build_arrow>
-    <static_arrow>OFF</static_arrow>
     <arrow_root>/usr/local</arrow_root>
+    <arrow.bfs.dir>${project.basedir}/../tools/build/arrow_install</arrow.bfs.dir>
+    <static_arrow>OFF</static_arrow>
     <build_protobuf>ON</build_protobuf>
     <build_jemalloc>ON</build_jemalloc>
```

Comment:
can you also disable jemalloc? it's not used actually

Reply:
Done